pkg/builder/nop/builder_test.go

```diff
@@ -45,8 +45,11 @@ func TestBasicFlow(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error executing OperationFromStatus: %v", err)
 	}
+	if bs.CreationTime.IsZero() {
+		t.Errorf("bs.CreationTime; want non-zero, got %v", bs.CreationTime)
+	}
 	if bs.StartTime.IsZero() {
-		t.Errorf("bs.StartTime; want non-zero, got %v", bs.StartTime)
+		t.Errorf("status.StartTime; want non-zero, got %v", bs.StartTime)
```

Comment:
Can you update the other error messages for consistency? Or return this to `bs.`